apps/web/app/(use-page-wrapper)/onboarding/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { redirect } from "next/navigation";
+
+import { FeaturesRepository } from "@calcom/features/flags/features.repository";
+import { prisma } from "@calcom/prisma";
+
+export async function OnboardingLayout({ children }: { children: React.ReactNode }) {
+  const featuresRepository = new FeaturesRepository(prisma);
+  const isOnboardingV3Enabled = await featuresRepository.checkIfFeatureIsEnabledGlobally("onboarding-v3");
+
+  if (!isOnboardingV3Enabled) {
+    redirect("/getting-started");
+  }
+
+  return <>{children}</>;
+}
```

Comment:
Pretty much ignore this entire PR of the onboarding-v3 flag isnt enabled in the insance